app/src/main/java/com/github/se/eventradar/viewmodel/EventsOverviewViewModel.kt

```diff
@@ -82,17 +113,41 @@ constructor(
   fun onViewListStatusChanged(state: MutableStateFlow<EventsOverviewUiState> = _uiState) {
     state.value = state.value.copy(viewList = !state.value.viewList)
   }
+
+  fun setFilterDialogOpen(isOpen: Boolean) {
```

Comment:
Just fyi, you don't need to change this because I've already told christine, but credits to [Jalen's comment](https://github.com/swent-sp-2024-event-radar/event-radar-app/pull/130#discussion_r1585952971) - this doesn't need to take in the boolean. Whenever it's called, we know the new value will be the opposite of what it is now.

Reply:
I believe Christine said that I would be keeping this method in this PR. Will edit.